core/pom.xml

```diff
@@ -82,17 +94,6 @@ THE SOFTWARE.
       <artifactId>crypto-util</artifactId>
       <version>1.1</version>
     </dependency>
-    <dependency>
-      <groupId>org.jvnet.hudson</groupId>
-      <artifactId>jtidy</artifactId>
```

Comment:
Looks like this is used in https://github.com/jenkinsci/nis-notification-lamp-plugin/blob/631e0dd7c76c6f04a0efa1c80c46040cf2e8bc37/src/main/java/jenkins/plugins/HTMLParser.java .

Reply:
Thanks for your feedback. It seems to be true dependencies end up becoming part of the public API.
> With a sufficient number of users of an API,  all observable behaviors of your system will be  depended on by somebody -- [The  Hyrums  Law]( http://www.hyrumslaw.com)